### PR TITLE
Added configuration file parsing

### LINKED ISSUE #6: Static list of topics to subscribe to
Until #5 is implemented and support for ROS graph introspection in `rclrs` is added, we will pass a list of topics and to subscribe to via a configuration file

### PR DESCRIPTION
**Pull request by esteve**
This PR adds parsing of configuration files where each  entry represents a mapping between topics, frame ids, ROS types and the converter function that will be used to convert between ROS and Rerun.

* Closes #6

### CODE REVIEW

**Review by esteve — COMMENTED** [review]
@jleibs could you have a look at this when you have a moment? I'd like to make sure this format represents well enough what you described in #10 Thanks.

**Comment by esteve** [comment]
@jleibs the format I'm using for the configuration file looks like this:

```toml

[[conversion]]
topic = "topic/foo"
frame_id = "frame1"
ros_type = "std_msgs/msg/String"
converter = "StringConverter"

[[conversion]]
topic = "topic/bar"
frame_id = "frame2"
ros_type = "std_msgs/msg/Int32"
converter = "Int32Converter"
```

File: src/config.rs [hunk 1]
```diff
@@ -0,0 +1,131 @@
+use anyhow::Result;
+use serde::Deserialize;
+use std::collections::HashMap;
+use std::fs;
+use std::path::Path;
+
+/// Represents a single conversion configuration.
+#[derive(Deserialize, Debug)]
+struct Conversion {
```
Comment (jleibs) [inline src/config.rs:9]: I think we need the ability to specify a destination Rerun entity here as well. Maybe with a behavior that if not-specified we default entity_path = topic.

File: src/config.rs [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    topic: String,
+    frame_id: String,
+    ros_type: String,
+    converter: String,
```
Comment (jleibs) [inline src/config.rs]: This is probably fine for now, but my intuition is that these parameters are redundant in some way.

Ideally we would have a registry of converters and each converter would know it's input ros_type.

If we only have a single converter for a given type, then we can infer it from the ros_type.

Alternatively, if you've specified a converter explicitly, then we should already know what the ros_type needs to be for it.

Comment (esteve) [reply src/config.rs]: Yeah, I agree. I ended up dropping the `converter` parameter in https://github.com/rerun-io/rerun-ros/pull/12 when I realized they were redundant.

File: src/config.rs (hunk 1 above)
Comment (esteve) [reply src/config.rs:9]: True, I hadn't thought about that, good catch

**Pull request merged**